src/http/errors.rs

```diff
@@ -0,0 +1,37 @@
+//! Various error responses.
+
+use hyper::{Body, Response, StatusCode};
+
+
+pub fn bad_request() -> Response<Body> {
+    Response::builder()
+    .status(StatusCode::BAD_REQUEST)
+    .header("Content-Type", "text/plain")
+    .body("Bad Request".into())
+    .unwrap()
+}
+
+pub fn initial_validation() -> Response<Body> {
+    Response::builder()
+    .status(StatusCode::SERVICE_UNAVAILABLE)
+    .header("Content-Type", "text/plain")
+    .body("Initial validation ongoing. Please wait.".into())
+    .unwrap()
+}
+
+pub fn method_not_allowed() -> Response<Body> {
+    Response::builder()
+    .status(StatusCode::METHOD_NOT_ALLOWED)
+    .header("Content-Type", "text/plain")
+    .body("Method Not Allowed".into())
+    .unwrap()
+}
+
+pub fn not_found() -> Response<Body> {
+    Response::builder()
+    .status(StatusCode::NOT_FOUND)
+    .header("Content-Type", "text/plain")
+    .body("Not Found".into())
+    .unwrap()
+}
+
```

Comment:
Don't know if this applicable here, but as an API consumer I prefer to have error responses also be of type 'application/json' and with the error message as JSON (well, except for 500).

OTOH probably everybody is dropping the whole thing after reading the HTTP Status code and never bother to parse the error message, also since it only re-iterates the error code.

Reply:
Hm, this is a bit tricky. I guess we could do all error messages under the `/api` tree as JSON? I kind of want to keep the ones where you might actually go with your browser as plain text.

Reply:
Under `/api` only makes sense, I guess

Reply:
After toying with possible solutions, I don’t think the effort is worth it. I think I’ll just leave it as it is.